src/tests/operate.test.js

```diff
@@ -0,0 +1,11 @@
+import operate from '../logic/operate';
+
+describe('operate', () => {
+  it('adds two numbers', () => {
+    expect(operate(2, 3, '+')).toBe('5');
+  });
+
+  it('multiple two numbers', () => {
+    expect(operate(2, 3, 'x')).toBe('6');
+  });
+});
```

Comment:
- [ ] Here happens the same, please make sure to test all functionalities and scenarios for the `operate.js` file! 🤓 🔧

Reply:
well resolved! ✔️